### ISSUE TITLE
AIR1 FUELING

### ISSUE THREAD

**Issue by Arywonfu** [echoroleplay/feedback#177]
**Description:** basically, fueling air1 is an absolute mess, I've found that the real only safe way to fuel the aircraft is to use a jerry can, and personally I don't think it should be a 50/50 chance on whether or not I blow up. 

Yesterday, I came in for a landing on the pad and not even touching the fuel pump or anything, the helicopter blew up. 

I also really think how fast air1 losses fuel should also be reconsidered. 

One last thing, the pump is also really extremely short and the only way to fuel is bringing the aircraft within inches from the pump. 

**Reproduction Steps:** Clearly describe the steps to reproduce the bug. 
1. Land air1 on the pad
2. Hope you don't blow up

**Expected Behavior:** sometimes I will blow up, and other times it will go smoothly. 

**Actual Behavior:** land and fuel the aircraft with no issues.

**Comment by Morphiushell** [comment]
How about making it when you land at a helli pad within the airport that you can fuel the helli from there

**Comment by Vtek79** [comment]
well then that should be the case for any helis then not just air 1 because right now renting helis come with half a tank or less and the fuel goes so fast u get less than 5 minutes of airtime with no real place to fuel them

**Comment by Britin123x** [comment]
Yeah, as Vtek said this is a heli scripting issue more than an Air1 issue. Having a more consistent fueling system for aircraft, especially with the new script, would make them used much more, as the hassle of dealing with it at least dissuades me from using it on a private level.

**Comment by stayingquietnow** [comment]
With the lack of places to fuel heli's now you just need to buy a bunch of jerrycans and land whereever safe to refuel. Air one fuel goes down as fast as private heli's too. You can't even make it to Cayo and back on one tank anymore
